ecs/src/nox/ecs/component/Sprite.cpp

```diff
@@ -0,0 +1,217 @@
+#include <nox/ecs/component/Sprite.h>
+
+#include <cassert>
```

Comment:
What are you asserting? Also, if you are asserting something, use the nox_assert.